packages/plugin-ext/src/plugin/command-registry.ts

```diff
@@ -197,19 +198,19 @@ export class CommandsConverter {
         // Existing code will have compiled against a non - optional version of the field, so asserting it to exist is ok
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
         return KnownCommands.map((external.command || external.id)!, external.arguments, (mappedId: string, mappedArgs: any[]) =>
-            ({
-                id: mappedId,
-                title: external.title || external.label || ' ',
-                tooltip: external.tooltip,
-                arguments: mappedArgs
-            }));
+        ({
+            id: mappedId,
+            title: external.title || external.label || ' ',
+            tooltip: external.tooltip,
+            arguments: mappedArgs
+        }));
     }
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     private executeSafeCommand<R>(...args: any[]): PromiseLike<R | undefined> {
         const command = this.commandsMap.get(args[0]);
         if (!command || !command.command) {
-            return Promise.reject('command NOT FOUND');
+            return Promise.reject(`command ${args[0]} not found`);
```

Comment:
`${args[0]}` this is just an internal command id which is a number increment. It won't give anything useful for user. vscode displays a plain string in this case:  https://github.com/microsoft/vscode/blob/3c0970f6fc793b6ec77dec542ce0d05c347be02f/src/vs/workbench/api/common/extHostCommands.ts#L355

Reply:
The user is not the target of what's written to the console log, the programmer is. The info might be useful for debugging purposes and it doesn't hurt writing it out.